requirements.txt

```
@@ -1,2 +1,3 @@
-librosa>=0.8.0
-jina~=2.2
\ No newline at end of file
+librosa
+docarray
```

fix the version for docarray as well
```suggestion
docarray
```